simcon_project/conversation_templates/urls/templates_urls.py

```diff
@@ -13,4 +13,6 @@
     path('new/', create_conversation_template_view, name="create-conversation-template-view"),
     path('edit/<uuid:pk>/', edit_conversation_template, name="edit_conversation_template"),
     path('redirect/template/', RedirectToTemplateCreation.as_view(), name="redirect-to-template-creation"),
+    path('display/archived/<show_archived>', update_cookie, name="display-archived-templates"),
```

Comment:
do we save it after its in the cookies? since people often clear those..

Reply:
What do you mean by it? Cookies or if the template is archived or unarchived?

Reply:
i hadnt tested this and was afraid we were only storing it in cookies and people delete this! but i tested my assumption meant to delete this comment